File: testcases/paper/Private/Private.cpp
class Private {
    int i, j;
public:
    void publicMethod() {
        i = 1;
    }
private:
    void privateMethod() {
        j = 2;
    }
};

// Private don't count: LCOM4 = 1
// Private count:       LCOM4 = 2

File: testcases/paper/Private/Private.go
package Private

type Private struct {
    i, j int
}

func (c Private) PublicMethod() {
    c.j = 2
}

func (c Private) privateMethod() {
    c.j = 2
}

// Private don't count: LCOM4 = 1 (outputs nothing)
// Private count:       LCOM4 = 2